README.md

```diff
@@ -106,3 +106,5 @@ in the online documentation for an overview on how to configure Spark.
 
 Please review the [Contribution to Spark guide](https://spark.apache.org/contributing.html)
 for information on how to get started contributing to the project.
+
+##Testing
```

Comment:
Testing is documented in https://spark.apache.org/developer-tools.html which is properly linked in build spark section. Let's don't duplicate the info here.